### PR TITLE
feat(connectors): describe BPMN Errors

### PR DESCRIPTION
**Pull request by tmetzke**
## What is the purpose of the change

Document how to throw BPMN Errors from Connectors.

closes https://github.com/camunda/connector-sdk/issues/199

## Are there related marketing activities

n/a

## When should this change go live?

After Connector SDK `0.3.0` was released. We'll ping here manually.

Current planned schedule: 15th November 

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.0/components/connectors/use-connectors.md [hunk 1]
```diff
@@ -20,4 +27,80 @@ Fields in the properties panel marked with an equals sign inside a circle indica
 
 ![feel Connectors](img/use-connectors-feel.png)
 
-Find the available Connectors in Camunda Platform 8 SaaS and how to use them in the [following pages](./out-of-the-box-connectors/available-connectors-overview.md).
+Each Connector defines its own set of properties that you can fill in. Find the details for Connectors provided by Camunda in the [out-of-the-box Connectors](./out-of-the-box-connectors/available-connectors-overview.md).
+
+## BPMN Errors
+
+Being able to deal with exceptional cases is a common requirement for business process models. Read more about our general best practices around this topic in [dealing with exceptions](/components/best-practices/development/dealing-with-problems-and-exceptions/).
```
Comment (tmetzke) [inline versioned_docs/version-8.0/components/connectors/use-connectors.md:34]:
```suggestion
Being able to deal with exceptional cases is a common requirement for business process models. Read more about our general best practices around this topic in [dealing with exceptions](/components/best-practices/development/dealing-with-problems-and-exceptions.md).
```

File: versioned_docs/version-8.1/components/connectors/use-connectors.md [hunk 2]
```diff
@@ -20,4 +27,80 @@ Fields in the properties panel marked with an equals sign inside a circle indica
 
 ![feel Connectors](img/use-connectors-feel.png)
 
-Find the available Connectors in Camunda Platform 8 SaaS and how to use them in the [following pages](./out-of-the-box-connectors/available-connectors-overview.md).
+Each Connector defines its own set of properties that you can fill in. Find the details for Connectors provided by Camunda in the [out-of-the-box Connectors](./out-of-the-box-connectors/available-connectors-overview.md).
+
+## BPMN Errors
+
+Being able to deal with exceptional cases is a common requirement for business process models. Read more about our general best practices around this topic in [dealing with exceptions](/components/best-practices/development/dealing-with-problems-and-exceptions/).
```
Comment (tmetzke) [inline versioned_docs/version-8.1/components/connectors/use-connectors.md:34]:
```suggestion
Being able to deal with exceptional cases is a common requirement for business process models. Read more about our general best practices around this topic in [dealing with exceptions](/components/best-practices/development/dealing-with-problems-and-exceptions.md).
```

**Review by christinaausley — DISMISSED** [review]
Let me know if you have questions about the adjustments in my latest commit 👍 I did adjust the descriptions as well alongside the best practices discussed [here](https://camunda.slack.com/archives/C026U8GBNSW/p1667580229027279).

File: docs/components/connectors/custom-built-connectors/connector-sdk.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 id: connector-sdk
 title: Connector SDK
-description: Introduction to the Connector SDK.
+description: Send messages to RabbitMQ from your BPMN process using the RabbitMQ Connector. The Connector SDK allows you to develop custom Connectors using Java code.
```
Comment (tmetzke) [inline docs/components/connectors/custom-built-connectors/connector-sdk.md:4]: I think you mixed up something here with another PR? 🙂

Comment (tmetzke) [inline docs/components/connectors/custom-built-connectors/connector-sdk.md:4]: @christinaausley , I think you mixed up something here with another PR? 🙂

Comment (tmetzke) [reply docs/components/connectors/custom-built-connectors/connector-sdk.md:4]:
```suggestion
description: The Connector SDK allows you to develop custom Connectors using Java code. Focus on the logic of the Connector, test it locally, and reuse its runtime logic..
```

**Comment by tmetzke** [comment]
Thanks @christinaausley, I like the new descriptions 👍

**Review by christinaausley — APPROVED** [review]

**Review by MoAtCamunda — APPROVED** [review]
No additional comments for now.

**Review by igpetrov — APPROVED** [review]

**Comment by tmetzke** [comment]
Merging this now to the `connector-sdk-0.3` branch that will cover all changes needed for SDK `0.3.0`. I'll create a PR for that branch, so we have one collective PR that needs to be held back until November 15th,

**Pull request merged**